bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/ExampleApp.kt

```diff
@@ -0,0 +1,16 @@
+package net.gini.android.bank.screen
+
+import androidx.multidex.MultiDexApplication
+import dagger.hilt.android.HiltAndroidApp
+
+
+/**
+ * Main Entry of the application.
+ */
+
+@HiltAndroidApp
+class ExampleApp : MultiDexApplication() {
+
```

Comment:
[Request] Please remove the empty lines 😉

Reply:
Done.